rocky/reports/views/aggregate_report.py

```diff
@@ -59,7 +59,13 @@ class LandingAggregateReportView(BreadcrumbsAggregateReportView, TemplateView):
     """
 
     def get(self, request: HttpRequest, *args: Any, **kwargs: Any) -> HttpResponse:
-        return redirect(reverse("aggregate_report_select_oois", kwargs=self.get_kwargs()))
+        kwargs = self.get_kwargs()
+        pre_selection = {
+            "clearance_level": ["1", "2", "3", "4"],
+            "clearance_type": "declared",
+        }
+        selection = self.get_selection(pre_selection)
+        return redirect(reverse("aggregate_report_select_oois", kwargs=kwargs) + selection)
```

Comment:
We can avoid string manipulations and add the selection as a dict to the kwargs as well

Reply:
Also I see that the default return value of `self.get_kwargs()` is just the organization code. Why is the organization a query parameter (as well)?

Reply:
The kwargs itself is not a query parameter, it needs the `organization_code` in `urls.py`. This is by design this way when we implemented the switch or orgs using this kwarg. The selection part will come after the kwarg.

Reply:
If we want to define a kwargs for oois selection is also possible, then in `urls.pu` we need to explicitely set a `<ooi_selection>` something like this so it can read it. We don't need to do this for query parameters.